src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ApiAttributesTest.php

```diff
@@ -815,33 +735,27 @@ public static function mapRequestPayloadProvider(): iterable
             'format' => 'json',
             'parameters' => ['comment' => 'Hello everyone!', 'approved' => '0'],
             'content' => null,
-            'expectedResponse' => <<<'JSON'
-                {
-                    "comment": "Hello everyone!",
-                    "approved": false
-                }
-                JSON,
+            'expectedResponse' => json_encode([
+                'comment' => 'Hello everyone!',
+                'approved' => false,
+            ]),
             'expectedStatusCode' => 200,
         ];
 
         yield 'malformed json request mapping non-nullable attribute without default value' => [
             'uri' => '/map-request-to-non-nullable-attribute-without-default-value.json',
             'format' => 'json',
             'parameters' => [],
-            'content' => <<<'JSON'
-                {
-                    "comment": "Hello everyone!",
-                    "approved": false,
-                }
-                JSON,
-            'expectedResponse' => <<<'JSON'
-                {
-                    "type": "https:\/\/tools.ietf.org\/html\/rfc2616#section-10",
-                    "title": "An error occurred",
-                    "status": 400,
-                    "detail": "Bad Request"
-                }
-                JSON,
+            'content' => json_encode([
+                'comment' => 'Hello everyone!',
+                'approved' => false,
+            ]),
+            'expectedResponse' => json_encode([
+                'type' => 'https://tools.ietf.org/html/rfc2616#section-10',
+                'title' => 'An error occurred',
+                'status' => 400,
+                'detail' => 'Bad Request',
+            ]),
```

Comment:
On my local machine, this test is now failing 🤔

Reply:
JSON is malformed in the test (trailing comma after `"approved": false`) so you cannot generate it using `json_encode`.